internal/gorson/io/io.go

```diff
@@ -212,38 +212,59 @@ func find(slice []*string, val *string) (int, bool) {
 }
 
 // deleteFromParameterStore deletes parameters at a given path from parameter store
-func deleteFromParameterStore(parameters []string, path util.ParameterStorePath, client ssmiface.SSMAPI) ([]string, error) {
+func deleteFromParameterStore(parameters []string, path util.ParameterStorePath, client ssmiface.SSMAPI) (deletedParams []string, err error) {
+	deletedParams = []string{}
+
 	fullPathParameters := make([]*string, len(parameters))
 	for idx, parameter := range parameters {
 		fullPathParameter := fmt.Sprintf("%s%s", path.String(), parameter)
 		fullPathParameters[idx] = &fullPathParameter
 	}
 
-	deleteParametersInput := ssm.DeleteParametersInput{
-		Names: fullPathParameters,
-	}
+	// DeleteParameters only works if the length of Names
```

Comment:
i wonder if it would be a bit cleaner to chuck the `fullPathParameters` into an array of arrays of size 10 first... that would make the iteration through the parameters a bit easier.

```
var chunkedParameters [][]string{}

for {
    if len(fullPathParameters) == 0 {
        break
    }
    
    if len(fullPathParameters) < chunkSize {
        chunkSize = len(fullPathParameters)
    }

    chunkedParameters = append(chunkedParameters, fullPathParameters[0:chunkSize])
    fullPathParameters = fullPathParameters[chunkSize:]
}

for _, params := range chunkedParameters {
    deleteParameterInput := ssm.DeleteParametersInput{
        Names: params
    }

    ...
}
```

Reply:
I think that's more straightforward than what I'm doing. I'll look into making that adjustment soon.